lb_logger.py

```diff
@@ -0,0 +1,9 @@
+import logging
+"""
+Instantiates a logger object with level set to INFO
+
+This will be used in all other files as the default logger object
+"""
+
+# logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
```

Comment:
it is beginning to look like all of the files that are being redirected to lb_logger file is only a one liner change. if we are updating the basicConfig inside Client.py, is there still a need for a centralized logger object?